.github/workflows/compile_engine.yml

```diff
@@ -0,0 +1,53 @@
+name: Release binaries for circus-engine
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          # - windows-latest
```

Comment:
Windows isn't working?

Reply:
It was not due to incompatibility of some library, but I might I have fixed that, will check

Reply:
So, turns out Opam is not able to install `gmp` or `pkgconf`, I'll mark that as a TODO in the issue #4